.eslintrc

```diff
@@ -0,0 +1,6 @@
+{
+  "extends" : "ship-components",
+   "rules": {
+      "indent": 0
```

Comment:
Indentation seems to be setup differently in different text editors.... I've noticed my tab space seems to be counted as 8 spaces and not a tab.